libs/block-editor/src/lib/extensions/dragHandler.extention.ts

```diff
@@ -143,7 +147,7 @@ export const DragHandler = (injector: Injector, resolver: ComponentFactoryResolv
                                     nodeToBeDragged = null;
                                     dragHandler.style.visibility = 'hidden';
                                 }
-                                return true;
+                                return false;
```

Comment:
What this does?

Reply:
don't interrupt the propagation of the event, so other extensions can react to the event.